pyproject.toml

```diff
@@ -39,14 +39,15 @@ classifiers = [
 ]
 requires-python = '>=3.7'
 dependencies = [
-    'pydantic>=2.0b3',
+    'pydantic>=2.0',
 ]
 dynamic = ['version']
 
 [project.optional-dependencies]
 all = [
     'phonenumbers>=8,<9',
     'pycountry>=22,<23',
+    'pandas==1.3.5',
```

Comment:
Why not `pandas>=2.0`?

Reply:
Because `pandas>=2.0` requires python >=3.8